pytheus/metrics.py

```diff
@@ -39,12 +51,24 @@ def __init__(
     def _validate_labels(self, labels: Sequence[str]):
         """
         Validates label names according to the regex.
-        Labels starting with `__` are reserved for internal use.
+        Labels starting with `__` are reserved for internal use by Prometheus.
         """
         for label in labels:
             if label.startswith('__') or label_name_re.fullmatch(label) is None:
                 raise ValueError(f'Invalid label name: {label}')
 
+    def collect(self) -> Sequence[Sample] | Iterator[Sample]:
+        """
+        Collects Samples for the metric.
+        If there are no required labels, will return a sequence with a single Sample.
```

Comment:
I assume no required labels means cardinality == 1 (Metric name, no labels), so that's why you return one item, right?

Reply:
Correct, if when you create a metric you do not pass required labels, you only have that one metric